src/store/Chat/index.js

```diff
@@ -0,0 +1,232 @@
+import API from '../endpoints';
+import makeDummy from '../dummy';
+
+// Flag to enable putting dummy data into redux
+// const DUMMY_DATA = process.env.NODE_ENV !== 'production';
+// const dummy = makeDummy(6, 50);
+
+const initialState = {
+  activeId: '',
+  activeIsOpen: null,
+  chats: [], // DUMMY_DATA ? dummy.chatSessions : [],
+  messages: [], // DUMMY_DATA ? dummy.messages : [],
+  operatorFilter: 'all',
+  config: {
+    apiServer: null,
+    operator: null,
+  },
+};
+
+
+// Constants
+//
+
+export const LOAD_CHATS_SUCCESS = 'CHAT_LOAD_CHATS_SUCCESS';
```

Comment:
If you're following the `ducks` pattern [I'm fairly certain](https://github.com/erikras/ducks-modular-redux) you don't need to export the constants.